source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt

```diff
@@ -63,11 +63,12 @@ Use a Consistent Sentence Structure
 Restrict Sentence Length
 ------------------------
 
-A well-written long sentence may be hard to follow and understand.
+Even a well-written long sentence may be hard to follow and understand.
```

Comment:
**question:** Does adding "Even" aid in the reader's understanding? I would posit that it's extraneous.

Reply:
I had to re-read this a few times. The juxtaposition of well-written and hard to follow and understand didn't make sense to me at first. An alternative re-write would be `A well-written long sentence may still be hard to follow and understand`. Otherwise why have `well-written` in there at all? We could just say `Long sentences may be hard to follow and understand` but I think the meaning we are aiming for is that even if the long sentence is well-written, we'd still prefer shorter sentences.

Reply:
**comment:** Keep "Even" then. I get the distinction now.